roqoqo-qasm/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "roqoqo-qasm"
-version = "0.4.6"
+version = "0.4.7"
```

Comment:
Actually we should go to version 0.5 here, because we are including larger changes